src/sort/index.ts

```diff
@@ -61,15 +62,20 @@ class Sort {
     }
   }
 
-  getPastPurchasesLabel(sort: Store.Sort) {
-    return sort.field;
+  getPastPurchasesLabel(sort: Store.Sort, index: number) {
```

Comment:
this and `getLabel` use the same logic, so move that logic into a function which they both call